tests/nested/core20/basic/task.yaml

```diff
@@ -31,3 +31,6 @@ execute: |
 
     echo "Ensure 'snap list' works and test-snapd-sh snap is removed"
     not execute_remote "snap list test-snapd-sh"
+
+    # make us fail so we get the boot logs for random reboot debugging
```

Comment:
if you run spread -shell google-nested:ubuntu-20.04-64:tests/nested/core20/basic then you can see the logs without exit 1

Reply:
right but you can't do that for a PR, I guess you could if you changed run-checks but that seemed like more work than I wanted to :-)